config/routes.rb

```diff
@@ -77,6 +77,8 @@
     collection do
       post "bulk-upload", to: "bulk_upload#bulk_upload"
       get "bulk-upload", to: "bulk_upload#show"
+      post "email-csv", to: "case_logs#email_csv"
```

Comment:
Is it worth splitting these out into their own controller (a la `bulk_upload`), rather than bolting onto `case_logs`?

Reply:
Does it need a new route at all? Is there a reason not to use the existing route/controller action based on format requested?

Reply:
I'm not sure I follow. Are you suggesting that the route should still be a request for `text/csv`? That seems like a weird choice if the outcome is redirecting them to an HTML page that tells them they'll receive an E-mail.

Reply:
Hmm no that is what I was suggesting, and when you put it that way it does seem a bit odd... I guess I was thinking of it like, what you're really asking the server for is still something in a text/csv format, the fact that it now happens to arrive via an email just being an implementation detail, but it's not _really_ correct I suppose...